.golangci.yml

```diff
@@ -19,7 +19,7 @@ linters-settings:
   revive:
     min-confidence: 0
   gocyclo:
-    min-complexity: 10
+    min-complexity: 11
```

Comment:
why is this required?

Reply:
Because it seems adding a new "if" in FirmwareInstallStatus makes it overflow the complexity